### ISSUE TITLE
Loss of Model Performance after Removing 'with torch.no_grad()' at Line 304 in akt.py

### ISSUE THREAD

**Issue by lif323** [arghosh/AKT#16]
Could you please explain the purpose of `wtih torch.no_grad()` at line 304 in akt.py? I removed this line and ran `python main.py --dataset assist2009_pid --model akt_pid`, but the model's performance degraded. The test AUC decreased from 0.826 to 0.824.
